lib/util/db.js

```diff
@@ -18,6 +18,45 @@ const { PartialPipe, mapStream } = require('../util/stream');
 const { construct } = require('../util/util');
 
 
+////////////////////////////////////////////////////////////////////////////////
+// DATABASE CONFIG
+
+const validateConfig = (config) => {
+  const { host, port, database, user, password, ssl, ...unsupported } = config;
+
+  if (!(ssl == null || ssl === true || ssl === false))
+    throw Problem.internal.invalidDatabaseConfig({ reason: 'ssl can only be true or false.' });
```

Comment:
Should I use a Problem even though it's not an HTTP response?

Reply:
It's good to use a Problem for anything user-facing, even if it's not an HTTP response.